HorizonEngine/Platform/OpenGL/GLWindow.h

```diff
@@ -42,6 +42,6 @@ namespace Hzn
 		GLFWwindow* m_Window;
 		bool m_Vsync = false;
 
-		std::unique_ptr<RenderContext> m_Context;
+		std::shared_ptr<RenderContext> m_Context;
```

Comment:
Why did you use shared_ptr instead of unique_ptr here

Reply:
Hi,
For now, the create methods for these classes return a reference counted pointer (shared_ptr), so I decided to change the fields in the App class to receive those.

Reply:
Oh okay.Thank you for resolving my doubts